utils/src/certificate.rs

```diff
@@ -2,53 +2,13 @@ use bech32::{Bech32, FromBase32, ToBase32};
 use chain_core::mempack::{ReadBuf, ReadError, Readable};
 use chain_core::property::Serialize;
 use chain_impl_mockchain::certificate::Certificate;
-use std::error::Error as StdError;
-use std::fmt;
 
-#[derive(Debug)]
-pub enum Error {
-    Bech32Error(bech32::Error),
-    HrpInvalid {
-        expected: &'static str,
-        actual: String,
-    },
-    ReadError(ReadError),
-}
-
-impl From<bech32::Error> for Error {
-    fn from(error: bech32::Error) -> Self {
-        Error::Bech32Error(error)
-    }
-}
+type StaticStr = &'static str;
```

Comment:
Eh, why obscure the common built-in type signature with a typedef?

Reply:
the `custom_error` macro does not like the raw version, this is a bit cir-convoluted way to go around the limitations.